SpatialGDK/Source/Private/EngineClasses/SpatialNetBitReader.cpp

```diff
@@ -53,7 +53,11 @@ FArchive& FSpatialNetBitReader::operator<<(UObject*& Value)
 		if (NetGUID.IsValid())
 		{
 			Value = PackageMapClient->GetObjectFromNetGUID(NetGUID, true);
-			checkf(Value, TEXT("An object ref %s should map to a valid object."), *ObjectRef.ToString());
+			if (!Value)
+			{
+				UE_LOG(LogTemp, Error, TEXT("An object ref %s %s should map to a valid object."), *ObjectRef.ToString(), ObjectRef.Path.IsSet() ? **ObjectRef.Path : TEXT("NO PATH"));
```

Comment:
Can you change from `LogTemp` to a proper category? Maybe also change to `Warning` and state that this object ref is going to resolve to `nullptr`?

Reply:
Done